README.md

```diff
@@ -42,17 +42,57 @@ setup dune-plot-style v00_02
 
 At this point you should be able to `#include "DUNEStyle.h"` or `from dunestyle import ...` as described in the following sections.
 
-#### Standalone Python setup
+### Standalone Python setup
+
+
 
 `dune-plot-style` supports being set up as a standalone Python package.
 You'll need to install a handful of common Python libraries for the examples to work.
 The recommended way to install these packages is to set up a virtual environment.
 This avoids potential package version conflicts and allows you to download the necessary packages on a remote server where you don't have root privileges, such as the GPVMs.
 
+##### Check Python version prerequisites
+
+**`dune-plot-style` requires a relatively recent version of Python3 (>= 3.9) to work.**
```

Comment:
Not sure whether I would call 3.9 (released in 2020) "relatively recent". Maybe just "... requires Python 3.9 to work."

Since Python 2 is end of life, and Python 3.9 is a release of Python 3, I don't think the explicit mention of "Python3" is necessary either.

Reply:
sounds good.  fixed up in 761b30f